Improve usability on README.md (#983)
spotipy-dev/spotipy: README.md

# Spotipy

##### Spotipy is a lightweight Python library for the [Spotify Web API](https://developer.spotify.com/documentation/web-api). With Spotipy you get full access to all of the music data provided by the Spotify platform.

![Tests](https://github.com/plamere/spotipy/workflows/Tests/badge.svg?branch=master) [![Documentation Status](https://readthedocs.org/projects/spotipy/badge/?version=latest)](https://spotipy.readthedocs.io/en/latest/?badge=latest)

## Table of Contents

- [Features](#features)
- [Documentation](#documentation)
- [Installation](#installation)
- [Quick Start](#quick-start)
- [Reporting Issues](#reporting-issues)
- [Contributing](#contributing)
- [License](#license)

## Features

Spotipy supports all of the features of the Spotify Web API including access to all end points, and support for user authorization. For details on the capabilities you are encouraged to review the [Spotify Web API](https://developer.spotify.com/web-api/) documentation.

## Documentation

> Spotipy's full documentation is online at [Spotipy Documentation](http://spotipy.readthedocs.org/). Some function may need a specific scope. Please visit https://developer.spotify.com/documentation/web-api/concepts/scopes to learn more. If you do not define the scope properly `ERROR 401 Unauthorized, permission missing` may occur.

Reviewer: ```suggestion
Spotipy's [full documentation is online](http://spotipy.readthedocs.org/). Some function may need a [specific scope](https://developer.spotify.com/documentation/web-api/concepts/scopes). If you do not define the scope properly `ERROR 401 Unauthorized, permission missing` may occur.
```

A full set of examples can be found in the [online documentation](http://spotipy.readthedocs.org/) and in the [Spotipy examples directory](https://github.com/plamere/spotipy/tree/master/examples).

> To get started, [install spotipy](#installation), create a new account or login on https://developers.spotify.com/. Go to the [dashboard](https://developer.spotify.com/dashboard), create an app and add your new ID and SECRET (ID and SECRET can be found on an app setting) to your environment ([step-by-step video](https://www.youtube.com/watch?v=kaBVN8uP358&t=0s)):

Reviewer: ```suggestion
To get started, [install spotipy](#installation), create a new account or log in on https://developers.spotify.com/. Go to the [dashboard](https://developer.spotify.com/dashboard), create an app and add your new ID and SECRET (ID and SECRET can be found on an app setting) to your environment ([step-by-step video](https://www.youtube.com/watch?v=kaBVN8uP358)):
```

> ### Without user authentication Example

Reviewer: ```suggestion
### Example without user authentication
```

for idx, track in enumerate(results['tracks']['items']):
    print(idx, track['name'])
```
Expected result:
```
0 Island In The Sun
1 Say It Ain't So
2 Buddy Holly
.
.
.
18 Troublemaker
19 Feels Like Summer
```

> ### With user authentication Example

Reviewer: ```suggestion
### Example with user authentication
```

If you have suggestions, bugs or other issues specific to this library,
file them [here](https://github.com/plamere/spotipy/issues).
Or just send a pull request.

## Contributing

If you are a developer with Python experience, and you would like to contribute to Spotipy, please be sure to follow the guidelines listed on documentation page

> > #### [Visit the guideline](https://spotipy.readthedocs.io/en/2.22.1/#contribute)

Reviewer: Don't link to a specific version, it will go out of date. Instead use the `latest` or `master` versions.

> ## License

Reviewer: I'm not sure if this section is needed, as GitHub has very recently added tabs at the top to link to the licence among others:

<img width="541" alt="image" src="https://github.com/spotipy-dev/spotipy/assets/1324225/aea289e3-fadf-4104-93dc-def2ab0ac4a6">
